src/reference/modules/administration/pages/uninstall.adoc

```diff
@@ -127,9 +127,15 @@ To delete a node from Rudder, you need to go to the node details page and click
 
 == Uninstalling Rudder server
 
+
 WARNING: Uninstalling the Rudder server package will remove configuration data,
-that will be lost if you don't have a backup. We advice to keep a backup of the server
-for future reference.
+that will be lost if you don't have a backup.
+
+In the general case, we advise to not uninstall Rudder server packages and to prefer to use a solution depending on your business and technical constraints where the underlying virtual machine is stopped, backupped, and hibernated. 
```

Comment:
what is the point of advising not to uninstall the server? It makes not much sense but should not be a problem (and could be common for relyays that may have other services running).

Reply:
keeping configuration and data. There is an awfull lot of configuration details, history, event logs, etc that are in a rudder server, the kind of things that you are likely to need if you migrate to a new Rudder or even to a new config solution. Just having archives make exploitation of these data very hard.

Reply:
I was only talking about the rudder root server here, not relays. The title should be clearer.